code/game/jobs/job_exp.dm

```diff
@@ -0,0 +1,237 @@
+
+// Player Verbs
+
+
+/client/verb/check_exp()
+	set name = "EXP"
+	set desc = "Shows your EXP status"
+	set category = "Special Verbs"
+	//if(!establish_db_connection())
+	//	return 0
+	var/body = "<html><head><title>EXP for [key]</title></head><BR>"
+	body += get_exp_report()
+	body += "</HTML>"
+	src << browse(body, "window=playerexp;size=550x615")
+
+
+// Admin Verbs
+
+/client/proc/cmd_mentor_check_player_exp()	//Allows mentors / admins to determine who the newer players are.
+	set category = "Admin"
+	set name = "Check Player XP"
+	if(!check_rights(R_MENTOR|R_MOD|R_ADMIN))
```

Comment:
Mentors should absolutely not have this, at all. They're not meant to have any tools other than their ability to answer mentorhelps. They're  a mentor, not administration.